<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="4" total="7" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-07-06T21:29:01 IST" name="Suite" finished-at="2025-07-06T21:30:29 IST" duration-ms="87552">
    <groups>
    </groups>
    <test started-at="2025-07-06T21:29:01 IST" name="Entrata Test" finished-at="2025-07-06T21:30:29 IST" duration-ms="87552">
      <class name="Entrata.testcases.ThirdcaseTest">
        <test-method signature="verifyDemoform(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.reflect.Method)[pri:3, instance:Entrata.testcases.ThirdcaseTest@38e79ae3]" started-at="2025-07-06T21:30:23 IST" name="verifyDemoform" data-provider="Formdata" finished-at="2025-07-06T21:30:27 IST" duration-ms="4462" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Philo]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Prakash]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[philo123@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Current]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[2147483647]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[101 - 300]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[Tester]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[a Resident]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[public void Entrata.testcases.ThirdcaseTest.verifyDemoform(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.reflect.Method) throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyDemoform -->
      </class> <!-- Entrata.testcases.ThirdcaseTest -->
      <class name="Entrata.testcases.SecondcaseTest">
        <test-method signature="verifyResourcespage(java.lang.reflect.Method)[pri:2, instance:Entrata.testcases.SecondcaseTest@6f45df59]" started-at="2025-07-06T21:29:32 IST" name="verifyResourcespage" finished-at="2025-07-06T21:30:21 IST" duration-ms="48791" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void Entrata.testcases.SecondcaseTest.verifyResourcespage(java.lang.reflect.Method) throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyResourcespage -->
      </class> <!-- Entrata.testcases.SecondcaseTest -->
      <class name="Entrata.testcases.FirstcaseTest">
        <test-method is-config="true" signature="launchDriver(java.lang.String)[pri:0, instance:Entrata.testcases.FirstcaseTest@4c309d4d]" started-at="2025-07-06T21:29:01 IST" name="launchDriver" finished-at="2025-07-06T21:29:27 IST" duration-ms="25888" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchDriver -->
        <test-method signature="verifyEntrataHomepage(java.lang.reflect.Method)[pri:1, instance:Entrata.testcases.FirstcaseTest@4c309d4d]" started-at="2025-07-06T21:29:27 IST" name="verifyEntrataHomepage" finished-at="2025-07-06T21:29:32 IST" duration-ms="4907" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void Entrata.testcases.FirstcaseTest.verifyEntrataHomepage(java.lang.reflect.Method)]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://www.entrata.com/press/blackstone] but found [https://www.entrata.com/]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://www.entrata.com/press/blackstone] but found [https://www.entrata.com/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at pages.firstcasepage.companyNews(firstcasepage.java:72)
at Entrata.testcases.FirstcaseTest.verifyEntrataHomepage(FirstcaseTest.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyEntrataHomepage -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:Entrata.testcases.FirstcaseTest@4c309d4d]" started-at="2025-07-06T21:30:27 IST" name="teardown" finished-at="2025-07-06T21:30:29 IST" duration-ms="1596" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- Entrata.testcases.FirstcaseTest -->
    </test> <!-- Entrata Test -->
  </suite> <!-- Suite -->
</testng-results>
